docs/guides/authoring/colorspaces.rst

```diff
@@ -0,0 +1,443 @@
+        description: |
+          lg16 : conversion from film log
+        to_reference: !<FileTransform> {src: lg16_to_lnf.spi1d, interpolation: nearest}
+
+
+First the ``lnf`` colorspace (short for linear float) is used as our
+reference colorspace. The name can be anything, but the idea of a
+reference colorspace is an important convention within OCIO: **all
+other colorspaces are defined as transforms either to or from this
+colorspace.**
+
+The ``lg16`` colorspace is a 16-bit log colorspace (see
+:ref:`config-spivfx` for an explanation of this colorspace). It has a
+name, a bitdepth and a description.
+
+The ``lg16`` colorspace is defined as a transform from ``lg16`` to the
+reference colorspace (``lnf``). That transform is to apply the LUT
+``lg16_to_lnf.spi1d``. This LUT has an input of ``lg16`` integers and
+outputs linear 32-bit float values
+
+Since the 1D LUT is automatically invertable by OCIO, we can use this
+colorspace both to convert ``lg16`` images to ``lnf``, and ``lnf``
+images to ``lg16``
+
+Importantly, because of the reference colorspace concept, we can
+convert images from ``lg16`` to the reference colorspace, and then on
+to any other colorspace.
+
+
+Here is another example colorspace, which is defined using
+``from_reference``.
+
+.. code-block:: yaml
+
+      - !<ColorSpace>
+        name: srgb8
+        bitdepth: 8ui
+        description: |
+          srgb8 :rgb display space for the srgb standard.
+        from_reference: !<FileTransform> {src: srgb8.spi3d, interpolation: linear}
+
+We use ``from_reference`` here because we have a LUT which transforms
+from the reference colorspace (``lnf`` in this example) to sRGB.
+
+In this case ``srgb8.spi3d`` is a complex 3D LUT which cannot be
+inverted, so it is considered a "display only" colorspace. If we did have a second 3D LUT to apply the inverse transform, we can specify both ``to_reference`` and ``from_reference``
+
+
+.. code-block:: yaml
+
+      - !<ColorSpace>
+        name: srgb8
+        bitdepth: 8ui
+        description: |
+          srgb8 :rgb display space for the srgb standard.
+        from_reference: !<FileTransform> {src: lnf_to_srgb8.spi3d, interpolation: linear}
+        to_reference: !<FileTransform> {src: srgb8_to_lnf.spi3d, interpolation: linear}
+
+Using multiple transforms
+-------------------------
+
+The previous example colorspaces all used a single transform each,
+however it is often useful to use multiple transforms to define a
+colorspace.
+
+.. code-block:: yaml
+
+      - !<ColorSpace>
+        name: srgb8
+        bitdepth: 8ui
+        description: |
+          srgb8 :rgb display space for the srgb standard.
+        from_reference: !<GroupTransform>
+          children:
+            - !<ColorSpaceTransform> {src: lnf, dst: lg16}
+            - !<FileTransform> {src: lg16_to_srgb8.spi3d, interpolation: linear}
+
+Here to get from the reference colorspace, we first use the
+``ColorSpaceTransform`` to convert from ``lnf`` to ``lg16``, then
```

Comment:
Eventually, we could ``:cpp:class:`` these class references to the API doc sections:

https://www.sphinx-doc.org/en/1.5.1/domains.html